src/metrics/_stats/responses/mod.rs

```diff
@@ -2,9 +2,13 @@ use serde_json::Value;
 use std::collections::HashMap;
 
 /// Nodes response
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Deserialize)]
 pub(crate) struct StatsResponse {
+    // Ignoring fields
+    #[serde(skip)]
     _shards: HashMap<String, Value>,
```

Comment:
What's the point of these fields if you're no longer going to deserialize them?

Reply:
no point :smile: removed :+1: